node/resolvers/catalog/calculatedAttachmentsResolver.ts

```diff
@@ -242,6 +242,10 @@ export default async (
     }),
   })
 
+  if (!reducedAttachmentSchema) {
+    return ''
```

Comment:
o retorno esperado é um JSON, nao? string vazia nao é um json válido, e vai dar erro quando for dar parse—poderia retornar um `{}`

nao sei se é o comportamento esperado ou nao, só quero saber se é essa a intenção

Reply:
Tá certo! Vou mudar o return pra ser um json vazio, e ter que mudar uma coisinha no product-customizer também pra fazer funcionar a validação se o schema existe.